pkg/image/registry/imagestreamimport/rest.go

```diff
@@ -448,7 +448,13 @@ func ensureSpecTag(stream *imageapi.ImageStream, tag, from string, importPolicy
 	zero := int64(0)
 	specTag.Generation = &zero
 	specTag.ImportPolicy = importPolicy
-	specTag.ReferencePolicy = referencePolicy
+
+	// Only set the reference policy if it's not already explicitly
+	// set on this tag.  Importing an image should not change the reference
+	// policy.
+	if len(specTag.ReferencePolicy.Type) == 0 {
+		specTag.ReferencePolicy = referencePolicy
+	}
```

Comment:
Does reference policy have a default value?

Reply:
Why does import policy behave differently?

Reply:
spec.tag.referencepolicy defaults to "source" also.  So they both behave the same in terms of their respective api objects.

The problem is that the import logic, when taking the inputs from the imagestreamimport resource, and applying them to the existing tag, was not respecting the existing reference policy value of the tag.

We have no way to know if:

1) the policy value on the tag was explicitly chosen by the user or defaulted
2) the policy value on the imagestreamimport request was explicitly chosen by the user or defaulted

So the best thing we can do, imho, is to not touch the value on the tag if it's set (which will always be true in any scenario in which the tag already exists at the time of the imagestreamimport request)